src/components/ui/PaymentRequestTable/PaymentRequestTable.tsx

```diff
@@ -23,68 +27,86 @@ const PaymentRequestTable = ({
   pageSize,
   totalRecords,
   onPaymentRequestClicked,
+  onPaymentRequestDuplicateClicked,
+  onPaymentRequestDeleteClicked,
+  onPaymentRequestCopyLinkClicked,
   onPageChanged,
   setStatusSortDirection,
   setCreatedSortDirection,
   setContactSortDirection,
   setAmountSortDirection,
 }: PaymentRequestTableProps) => {
   return (
-    <table className="table-fixed text-left w-full">
-      <thead>
-        <tr>
-          <th className={classNames(commonThClasses, 'w-44 text-left')}>
-            <ColumnHeader
-              label="Status"
-              onSort={(sortDirection) => setStatusSortDirection && setStatusSortDirection(sortDirection)}
-            />
-          </th>
-          <th className={classNames(commonThClasses, 'w-44 text-left')}>
-            <ColumnHeader
-              label="Created"
-              onSort={(sortDirection) => setCreatedSortDirection && setCreatedSortDirection(sortDirection)}
-            />
-          </th>
-          <th className={classNames(commonThClasses, 'w-44 text-left')}>
-            <ColumnHeader
-              label="Contact"
-              onSort={(sortDirection) => setContactSortDirection && setContactSortDirection(sortDirection)}
-            />
-          </th>
-          <th className={classNames(commonThClasses, 'w-44 text-right')}>
-            <ColumnHeader
-              label="Amount"
-              onSort={(sortDirection) => setAmountSortDirection && setAmountSortDirection(sortDirection)}
-            />
-          </th>
+    <>
+      <table className="table-fixed text-left w-full">
+        <colgroup>
+          <col />
+          <col />
+          <col />
+          <col />
+          <col />
+          <col />
+          <col className="w-8" />
+        </colgroup>
```

Comment:
It is, because we need to set the width of the last column (the one with the action menu) and, as we're using a table header with colspan 2, there's no other way to set it.